### DISCUSSION TITLE
Coding for novices

### DISCUSSION THREAD

**Discussion by daedaesplayground1991** [community/community#22332] [New to GitHub]
<p>What are some of the easiest and most effective ways for a begginer to learn to code, plus learn the concepts behind it as well.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hey <a class="mention" href="/daedaesplayground199">@daedaesplayground199</a>,</p>
<p>Great work on taking the leap into coding! Personally, I find pages or tutorials with examples super good for learning new tech. <a href="https://www.w3schools.com/" rel="nofollow noopener">W3Schools</a> and <a href="https://developer.mozilla.org/" rel="nofollow noopener">MDN (Mozilla Developer Network)</a> are both great for web development topics, I’ve found. <a href="https://lab.github.com/" rel="nofollow noopener">GitHub Learning Lab</a> is also good for learning Git/GitHub.</p>
<p>It really depends how you learn and what language you’re learning. If you’re more of a learning-through-listening person, video tutorials such as ones from <a href="https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ" rel="nofollow noopener">freeCodeCamp</a> might be a good move. If you like reading, blog posts from various sites (like <a href="https://bitsofco.de/" rel="nofollow noopener">bitsofcode</a>) about topics offer a lot of description to make you pro at specific topics in any time :) </p>

**Comment by Seth666tl** [comment]
<p>Hello , i am trying to learn how to create a website by code and i want to find myself the right way to do it .</p>
<p>My question is, how do you write you’re code? what do you do first ?<br>
The head, content, footer ?<br>
can you please give me some examples?</p>
<p>I learn the basics of HTML, CSS a bit of PHP for WordPress, at this moment i can do a website by code as a beginner</p>

**Comment by ghost** [comment]
<p>Hey there <a class="mention" href="/seth666tl">@seth666tl</a>,</p>
<p>If you’re familiar with HTML basics, you’ll be more than capable with getting started! In terms of a general page layout, this is how HTML is structured: <a href="https://gist.github.com/ItsPugle/4df97d2fe3cb672f6dc07d8edcbcf549" rel="nofollow noopener">https://gist.github.com/ItsPugle/4df97d2fe3cb672f6dc07d8edcbcf549</a></p>
<p>The landing page should always be on the index.html file. How you go about writing it is up to you, however, I personally start with the general structure and head tags so I can test the site as I go. After, I then add content, styles (using a CSS stylesheet) and any JS.</p>
<p>Does this get you going in the right direction?</p>
